sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/producer/MessageProducer.java

```diff
@@ -22,4 +22,6 @@
   Future<RecordMetadata> send(K key, V value);
 
   Future<RecordMetadata> send(K key, V value, Headers headers);
+
+  void flush();
```

Comment:
A blank implementation would work for making it non breaking, not forcing the implementing class to adapt, which is nice. 
I think personally I kind of dislike a blank implementation because it might suggest that there is some function behind the method, I am not super obviously made aware of it being a blank.

Reply:
And for the default implementation I wonder if there is anything to do...We can not reference fields like the producer in the interface and we do not return anything..So at most we could throw a UnsupportedOperationException or similiar

Reply:
Imho, we can not merge it with a breaking change, but have to wait for a new major version. So a blank implementation would work for me and has the same functionality as before.